.github/workflows/installers-conda.yml

```diff
@@ -279,4 +290,4 @@ jobs:
           GITHUB_TOKEN: ${{ github.token }}
         with:
           upload_url: ${{ steps.get_release.outputs.upload_url }}
-          asset_path: ${{ env.PKG_FILE }}
+          asset_path: ${{ env.PKG_GLOB }}*
```

Comment:
Does this step upload now both the installer and `sha256sum.txt` file?

Reply:
Yes, both the installer and hash file are uploaded. When uploading artifacts on "Build as release candidate", it will appear as one artifact but it is really a zip file containing both the installer and the hash file; see [this test workflow dispatch](https://github.com/mrclary/spyder/actions/runs/4259084419). When uploading release assets on release event, both the installer and text file will appear as separate assets; see [this test release](https://github.com/mrclary/spyder/releases/tag/v6.0.0a1).